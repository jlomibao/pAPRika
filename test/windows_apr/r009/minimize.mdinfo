

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6895E-01     1.5986E+00     H14        51

 BOND    =        1.1108  ANGLE   =      107.7371  DIHED      =       51.4538
 VDWAALS =      -46.2346  EEL     =     1337.5369  EGB        =     -135.9263
 1-4 VDW =        5.2047  1-4 EEL =    -2132.1889  RESTRAINT  =        0.0006
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================
