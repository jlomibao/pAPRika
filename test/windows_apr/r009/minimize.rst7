defa
  125
  -0.5217315   4.9759898   1.8352757   1.0157616   4.9108597   1.8351998
  -0.8855281   4.2791366   0.6084446  -0.8854211   4.2789355   3.0620281
  -0.9016866   6.0041298   1.8353693   1.4813128   5.9031800   1.8352544
   1.3195431   4.1856008   0.6086052   1.3196431   4.1853984   3.0616505
   2.6868566   4.0833479   0.1137419   2.6869866   4.0831414   3.5564113
   0.2013740   3.8585041   3.7830409   0.2012073   3.8588667  -0.1127547
   0.1802997   3.3488431   4.8904007   0.1800368   3.3494714  -1.2202183
   2.6633132   4.0470039  -0.9815032   3.2459825   4.9656465   0.4459640
   2.6635113   4.0466645   4.6516534   3.2460723   4.9654915   3.2242584
  -4.6360085   1.9609975   1.8353959  -3.8101884   3.2595769   1.8353913
  -4.2146787   1.2972057   0.6086272  -4.2146617   1.2971776   3.0621492
  -5.7162904   2.1465470   1.8354032  -4.4363654   4.1592101   1.8354228
  -3.0303963   3.1595569   0.6085803  -3.0303566   3.1595063   3.0621827
  -2.2566394   4.2914967   0.1136677  -2.2564569   4.2913833   3.5570085
  -3.3068009   2.0275522   3.7832789  -3.3068647   2.0276183  -0.1125204
  -2.8758559   1.7539595   4.8904801  -2.8760172   1.7540933  -1.2197585
  -2.2364295   4.2526544  -0.9815582  -2.7400319   5.2175743   0.4455652
  -2.2360889   4.2525335   4.6522312  -2.7398352   5.2174954   3.2251867
  -4.0834624  -3.1099034   1.8352300  -4.7950445  -1.7453630   1.8352898
  -3.2980045  -3.0769788   0.6084140  -3.2978231  -3.0769875   3.0619341
  -4.7843590  -3.9526261   1.8352734  -5.8872479  -1.8377168   1.8353328
  -4.3184760  -1.1200722   0.6084856  -4.3183517  -1.1201075   3.0620666
  -4.9114645   0.1162941   0.1138299  -4.9113797   0.1161869   3.5568636
  -3.4762840  -1.9255680   3.7830941  -3.4765457  -1.9255408  -0.1126897
  -3.0237639  -1.6892029   4.8902591  -3.0242174  -1.6891783  -1.2199181
  -4.8678545   0.1145069  -0.9814016  -5.9551010   0.1610918   0.4458844
  -4.8677545   0.1142917   4.6520943  -5.9550255   0.1609702   3.2248366
   0.5850522  -5.1654852   1.8348868  -0.9525018  -5.1002415   1.8349972
   0.9488701  -4.4684924   0.6081124   0.9490229  -4.4685944   3.0616721
   0.9649465  -6.1936476   1.8348112  -1.4181166  -6.0925320   1.8349955
  -1.2561467  -4.3749516   0.6082552  -1.2559854  -4.3750094   3.0618181
  -2.6234630  -4.2708011   0.1137626  -2.6232414  -4.2708337   3.5564782
  -0.1375617  -4.0480136   3.7828699  -0.1378190  -4.0479105  -0.1129263
  -0.1162311  -3.5380033   4.8900696  -0.1166417  -3.5379033  -1.2201158
  -2.6002767  -4.2341688  -0.9814806  -3.1837860  -5.1522939   0.4461068
  -2.5999189  -4.2342178   4.6517190  -3.1836269  -5.1523092   3.2241935
   4.6985733  -2.1491216   1.8347278   3.8734352  -3.4481367   1.8347323
   4.2768294  -1.4854598   0.6080307   4.2769228  -1.4855246   3.0614899
   5.7789495  -2.3341211   1.8346786   4.5000549  -4.3474616   1.8346868
   3.0935099  -3.3484394   0.6079879   3.0936033  -3.3484902   3.0615415
   2.3199690  -4.4806250   0.1132942   2.3201919  -4.4807407   3.5562743
   3.3694966  -2.2164066   3.7826516   3.3693892  -2.2163443  -0.1131112
   2.9385196  -1.9430812   4.8899054   2.9384093  -1.9430240  -1.2203500
   2.2996828  -4.4419263  -0.9819354   2.8035573  -5.4065626   0.4452974
   2.3000859  -4.4421449   4.6515109   2.8037516  -5.4066314   3.2240987
   4.1487992   2.9227313   1.8349520   4.8584230   1.5570318   1.8348486
   3.3640918   2.8906251   0.6075427   3.3641951   2.8904879   3.0624275
   4.8510214   3.7643498   1.8349636   5.9507603   1.6477887   1.8347933
   4.3808414   0.9320343   0.6082301   4.3809607   0.9319252   3.0614604
   4.9732905  -0.3044214   0.1130556   4.9734759  -0.3045534   3.5564703
   3.5394364   1.7379200   3.7825015   3.5393005   1.7381384  -0.1126761
   3.0854079   1.5011428   4.8889613   3.0852637   1.5015195  -1.2191504
   4.9290776  -0.3024420  -0.9821514   6.0171335  -0.3493639   0.4444405
   4.9294171  -0.3026560   4.6516836   6.0172696  -0.3494813   3.2249280
   0.0002950   0.0068113  17.9514974  -0.9137582  -0.5719898  17.7706748
   0.8630748  -0.6459475  17.7706177   0.0343668   0.8259870  17.2228067
   0.0234321   0.5616693  19.3892275   0.9307801   1.1619143  19.5293037
  -0.0232225  -0.5614415  20.4474738  -0.8309403   1.2351688  19.5293773
  -0.9305300  -1.1617259  20.3073034   0.8311855  -1.2348943  20.3073166
  -0.0001951  -0.0066758  21.8852537  -0.8630207   0.6460254  22.0661044
  -0.0342712  -0.8259130  22.6138700   0.9138195   0.5721620  22.0661499
  -0.0000762   0.0001535  -6.0005055  -0.0001768  -0.0000621  -9.0000704
   0.0001751   2.2000035 -11.2000237
