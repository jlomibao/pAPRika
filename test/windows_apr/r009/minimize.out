
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 19:01:22

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/r009
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   380773 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9698E+02     1.0074E+01     2.4798E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.6645  EEL     =     1326.5993  EGB        =     -129.6177
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -796.9808
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1131E+02     4.6904E-01     1.5985E+00     H14        51

 BOND    =        1.1090  ANGLE   =      107.7361  DIHED      =       51.4561
 VDWAALS =      -46.2301  EEL     =     1337.4841  EGB        =     -135.9130
 1-4 VDW =        5.2024  1-4 EEL =    -2132.1510  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1131E+02     4.6902E-01     1.5985E+00     H14        51

 BOND    =        1.1092  ANGLE   =      107.7363  DIHED      =       51.4558
 VDWAALS =      -46.2306  EEL     =     1337.4903  EGB        =     -135.9146
 1-4 VDW =        5.2027  1-4 EEL =    -2132.1557  RESTRAINT  =        0.0006
 EAMBER  =     -811.3066
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1131E+02     4.6902E-01     1.5985E+00     H14        51

 BOND    =        1.1094  ANGLE   =      107.7364  DIHED      =       51.4556
 VDWAALS =      -46.2311  EEL     =     1337.4961  EGB        =     -135.9161
 1-4 VDW =        5.2029  1-4 EEL =    -2132.1598  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1131E+02     4.6901E-01     1.5985E+00     H14        51

 BOND    =        1.1096  ANGLE   =      107.7364  DIHED      =       51.4553
 VDWAALS =      -46.2316  EEL     =     1337.5019  EGB        =     -135.9175
 1-4 VDW =        5.2032  1-4 EEL =    -2132.1639  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1131E+02     4.6900E-01     1.5985E+00     H14        51

 BOND    =        1.1098  ANGLE   =      107.7365  DIHED      =       51.4551
 VDWAALS =      -46.2321  EEL     =     1337.5077  EGB        =     -135.9190
 1-4 VDW =        5.2034  1-4 EEL =    -2132.1680  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1131E+02     4.6898E-01     1.5986E+00     H14        51

 BOND    =        1.1100  ANGLE   =      107.7367  DIHED      =       51.4548
 VDWAALS =      -46.2326  EEL     =     1337.5136  EGB        =     -135.9204
 1-4 VDW =        5.2037  1-4 EEL =    -2132.1723  RESTRAINT  =        0.0006
 EAMBER  =     -811.3066
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1131E+02     4.6898E-01     1.5986E+00     H14        51

 BOND    =        1.1102  ANGLE   =      107.7368  DIHED      =       51.4546
 VDWAALS =      -46.2331  EEL     =     1337.5195  EGB        =     -135.9220
 1-4 VDW =        5.2040  1-4 EEL =    -2132.1764  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1131E+02     4.6897E-01     1.5986E+00     H14        51

 BOND    =        1.1104  ANGLE   =      107.7369  DIHED      =       51.4543
 VDWAALS =      -46.2336  EEL     =     1337.5253  EGB        =     -135.9234
 1-4 VDW =        5.2042  1-4 EEL =    -2132.1806  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1131E+02     4.6896E-01     1.5986E+00     H14        51

 BOND    =        1.1106  ANGLE   =      107.7370  DIHED      =       51.4541
 VDWAALS =      -46.2341  EEL     =     1337.5309  EGB        =     -135.9248
 1-4 VDW =        5.2045  1-4 EEL =    -2132.1846  RESTRAINT  =        0.0006
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6895E-01     1.5986E+00     H14        51

 BOND    =        1.1108  ANGLE   =      107.7371  DIHED      =       51.4538
 VDWAALS =      -46.2346  EEL     =     1337.5369  EGB        =     -135.9263
 1-4 VDW =        5.2047  1-4 EEL =    -2132.1889  RESTRAINT  =        0.0006
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1131E+02     4.6895E-01     1.5986E+00     H14        51

 BOND    =        1.1108  ANGLE   =      107.7371  DIHED      =       51.4538
 VDWAALS =      -46.2346  EEL     =     1337.5369  EGB        =     -135.9263
 1-4 VDW =        5.2047  1-4 EEL =    -2132.1889  RESTRAINT  =        0.0006
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.303    9.302    0.001    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.387  147.398    0.011    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.472   49.483    0.011    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.000   62.003    0.002    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.189 -146.178    0.011    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.543  -72.548    0.006    0.000 t
  DUM  DM1    3 --  C    BUT    2:   23.952   24.000    0.048    0.000 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.988  180.000    0.012    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.787  180.000    0.213    0.000 a
                                       Total distance penalty:      0.000
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.472
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.14 (21.45% of Gen B)
|                Calc gb diag               3.88 (38.93% of Gen B)
|                Calc gb off-diag           3.94 (39.59% of Gen B)
|                Other                      0.00 ( 0.03% of Gen B)
|             Gen Born time              9.96 (100.0% of Nonbo)
|          Nonbond force              9.96 (90.33% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.35% of Force)
|          Other                      0.04 ( 0.32% of Force)
|       Force time                11.02 (100.0% of Runmd)
|    Runmd Time                11.02 (99.81% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.04 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 19:01:22.377  on 04/10/2019
|           Setup done at 19:01:22.382  on 04/10/2019
|           Run   done at 19:01:33.421  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
